src/ui/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,102 @@
+import { forwardRef } from "react";
+import styled from "styled-components";
+import shapesPink1Url from "./shapes-pink-1.svg";
+import shapesPink2Url from "./shapes-pink-2.svg";
+import shapesBlue1Url from "./shapes-blue-1.svg";
+import { H1, Subheading1, P } from "../Typography";
+
+type BackgroundType = "pink-1" | "pink-2" | "blue-1";
+
+const Wrap = styled.div<{
+  $backgroundType: BackgroundType;
+}>`
+  padding: 80px 0 80px 0;
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  justify-content: space-evenly;
+  color: var(--color-white);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+
+  ${(props) =>
+    props.$backgroundType === "pink-1" &&
+    `
+    background-image: url(${shapesPink1Url});
+    background-color: var(--color-red);
+  `}
+
+  ${(props) =>
+    props.$backgroundType === "pink-2" &&
+    `
+    background-image: url(${shapesPink2Url});
+    background-color: var(--color-red);
+  `}
+
+  ${(props) =>
+    props.$backgroundType === "blue-1" &&
+    `
+    background-image: url(${shapesBlue1Url});
+    background-color: var(--color-blue);
+  `}
+`;
+
+const BlockHeader = styled.div`
+  flex-grow: 60;
+  flex-basis: 0;
+`;
+
+const BlockSpacer = styled.div`
+  flex-grow: 40;
+  flex-basis: 0;
+`;
```

Comment:
normally it's good practice to use simplified ratio in flex grow/shrink, so in this case header gets grow:3, spacer gets grow:2